website/content/docs/api-gateway/upgrade-specific-versions.mdx

```diff
@@ -9,11 +9,166 @@ description: >-
 
 This topic describes how to upgrade Consul API Gateway.
 
-## Breaking Changes
+
+## v0.3.0
+
+Consul API Gateway v0.3.0 introduces a breaking change for people upgrading from lower versions. Gateways with `listner` with a `certificateRef` defined in a different namespace now require a [`ReferencePolicy`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy) that explicitly allows `Gateways` from the gateway's namesapce to use `certificateRef` in the `certificateRef`'s namespace.
```

Comment:
```suggestion
Consul API Gateway v0.3.0 introduces a breaking change. Gateways with `listner` with a `certificateRef` defined in a different namespace now require a [`ReferencePolicy`](https://gateway-api.sigs.k8s.io/v1alpha2/references/spec/#gateway.networking.k8s.io/v1alpha2.ReferencePolicy) that explicitly allows `Gateways` from the gateway's namesapce to use `certificateRef` in the `certificateRef`'s namespace.
```
Aren't all upgrades from a lower version?
Also, is it `certificateRef` or `certificateRefs`?